### PR TITLE
add overlay for samsung tab a7 lite wifi

### PR DESCRIPTION
**Pull request by youheng7185**

### CODE REVIEW

File: Samsung/gta7litewifi/res/xml/power_profile.xml [hunk 1]
```diff
@@ -0,0 +1,67 @@
...
+        <value>400000</value>
+    </array>
+    <array name="cpu.active.cluster0">
+        <value>0.1</value>
+    </array>
+    <item name="cpu.idle">0.1</item>
+    <array name="memory.bandwidths">
+        <value>22.7</value>
+    </array>
+    <item name="battery.capacity">1000</item>
```
Comment (phhusson) [inline Samsung/gta7litewifi/res/xml/power_profile.xml]: Nope.

**Review by phhusson — CHANGES_REQUESTED** [review]
Overlay contains obviously wrong values

File: Samsung/gta7litewifi/res/xml/power_profile.xml (hunk 1 above)
Comment (teerasak) [reply Samsung/gta7litewifi/res/xml/power_profile.xml]: Please add for SM-T225 ( Samsung tab a7 LTE ) also.

Comment (youheng7185) [reply Samsung/gta7litewifi/res/xml/power_profile.xml]: hi, i already replaced with the correct power profile.xml have a look

**Review by phhusson — APPROVED** [review]

**Pull request merged**